inginious/frontend/i18n/es/LC_MESSAGES/messages.po

```diff
@@ -527,6 +527,14 @@ msgstr "Evaluación de las tareas (con éxito)"
 msgid "Evaluation submissions (Failed)"
 msgstr "Evaluación de las tareas (han fallado)"
 
+#: inginious/frontend/pages/course_admin/student_list.py:40
+msgid "Username was not found with an already existing account in UNCode"
+msgstr ""
+
+#: inginious/frontend/pages/course_admin/student_list.py:44
+msgid "User could not be registered due to internal server error"
+msgstr ""
```

Comment:
Please include spanish translations.

Reply:
I manually included the spanish translations